src/radical/pilot/agent/agent_0.py

```diff
@@ -340,6 +347,12 @@ def _start_services(self):
         If a `./services` file exist, reserve a compute node and run that file
         there as bash script.
         '''
+        # TODO
+        #  there is no pilot description/pilot instance accessible here. how to get services list
```

Comment:
The pilot gets it's configuration from the pilot launcher.  Currently service commands are passed as arguments to the bootstrapper (see [here](https://github.com/radical-cybertools/radical.pilot/blob/devel/src/radical/pilot/pmgr/launching/base.py#L851)).  That should be changed to add them to the `agent_config` (same file, 5 lines below, the agent config is created).  If that happens, then the service descriptions will be available here as `self._cfg.services`.

Reply:
done